src/main/java/lotto/controller/LottoController.java

```diff
@@ -13,13 +13,18 @@ public class LottoController {
     public static void main(String[] args) {
         int totalPriceToBuyLotto = InputView.totalPrice();
         int numberOfLotto = totalPriceToBuyLotto / 1000;
-        OutputView.numberOfLotto(numberOfLotto);
-        List<Lotto> allLotto = Lotto.lottoFactory(numberOfLotto);
+        int manualLottoCount = InputView.manualLottoCount();
+        List<Lotto> allLotto = Lotto.lottoFactoryForManual(InputView.manualLottos(manualLottoCount));
+
+        int autoLottoCount = numberOfLotto - manualLottoCount;
+        OutputView.numberOfLotto(manualLottoCount, autoLottoCount);
+        allLotto.addAll(Lotto.lottoFactoryForAuto(autoLottoCount));
+        System.out.println(allLotto.size());
         OutputView.allLotto(allLotto);
 
         Lotto winningNumbers = new Lotto(InputView.winningNumber());
```

Comment:
```suggestion
        Lotto winningNumbers = InputView.winningNumber();
```
View에서 바로 Lotto 객체로 반환해도 되지 않을까?